tests/Functional/ExportActionsTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineORMAdminBundle\Tests\Functional;
+
+use Symfony\Component\HttpFoundation\Request;
+
+final class ExportActionsTest extends BaseFunctionalTestCase
+{
+    /**
+     * @dataProvider provideExportActionCases
+     *
+     * @param array<mixed> $parameters
+     * @param array<mixed> $expected
+     */
+    public function testExportAction(string $url, array $parameters, array $expected): void
+    {
+        ob_start();
+        $this->client->request(Request::METHOD_GET, $url, $parameters);
+        $content = ob_get_contents();
+        ob_end_clean();
+
+        self::assertResponseIsSuccessful();
+        static::assertSame(
+            json_encode($expected, \JSON_THROW_ON_ERROR),
+            '' !== $content ? $content : $this->client->getInternalResponse()->getContent()
```

Comment:
have to do this hack, depends on the php version it works with the ob_get_contents or not.

Reply:
Can you add a comment to explain the php version we need `ob_get_contents` and the one we don't need it ?
This way we'll know when to remove the hack.

Reply:
Yes, I will add a reminder here, I think it starts changing behavior on php 8.0, but not completely sure, will have to try.

Reply:
I have added a note about this, the problem was Symfony < 6.2.